src/main/java/com/ververica/flink/table/gateway/SqlCommandParser.java

```diff
@@ -27,35 +51,142 @@
 
 /**
  * Simple parser for determining the type of command and its parameters.
- * TODO refactor using flink-sql-parser
  */
 public final class SqlCommandParser {
 
 	private SqlCommandParser() {
 		// private
 	}
 
-	public static Optional<SqlCommandCall> parse(String stmt) {
+	/**
+	 * Parse the given statement and return corresponding SqlCommandCall.
+	 *
+	 * <p>only `show current_catalog`, `show current_database`, `show modules`, `set`, `reset`,
```

Comment:
show current catalog, show current database (without underscore)

Reply:
flink sql parser does not support `show current catalog` and `show current database`